extensions/cornerstone/src/commandsModule.ts

```diff
@@ -72,7 +96,129 @@ const commandsModule = ({ servicesManager }) => {
     return toolGroup;
   }
 
+  function _showViewerContextMenu(viewerElement, options) {
+    let defaultPointsPosition = [];
+    if (options.nearbyToolData) {
+      defaultPointsPosition = commandsManager.runCommand(
+        'getToolDataActiveCanvasPoints',
+        { toolData: options.nearbyToolData }
+      );
+    }
```

Comment:
can't you just have all this inside the `showViewerContextMenu`, it is refactored, but again this refactoring doesn't do anything useful other than making the reader jump lines to find what the function is doing

Reply:
No, the point is that showViewerContextMenu sets up the conditions to show a generic context menu, while the showContextMenu actually displays a context menu.  As soon as there are other types of menus with different check conditions, they will need their own entry points.  Eventually the showContextMenu probably needs to move into @ohif/ui as it is not cornerstone specific.
showViewerContextMenu only works for CS3D viewports.